company/remote/index.md

```diff
@@ -8,6 +8,35 @@ Sourcegraph is an all-remote company, which means:
   - Teammates are responsible for setting their own work hours and marking them in Google Calendar.
   - We try to limit the number of synchronous meetings, but there are some [company](../../handbook/communication/company_meeting.md) and team meetings that you need to attend which might be outside of your normal working hours. We do our best to optimize meeting times for all involved participants.
 
+## FAQ
+
+### How are you able to maintain team cohesiveness and togetherness while working remotely?
```

Comment:
"cohesiveness and togetherness" sounds awkward. "team spirit", "team culture", "team cohesion", etc., all sound better IMO.

Reply:
```suggestion
### How are you able to maintain team cohesion while working remotely?
```